sycl/doc/GetStartedGuide.md

```diff
@@ -166,10 +166,10 @@ To enable support for ROCm devices, follow the instructions for the Linux
 DPC++ toolchain, but add the `--rocm` flag to `configure.py`
 
 Enabling this flag requires an installation of
-ROCm 4.1.0 on the system, refer to
+ROCm 4.2.0 on the system, refer to
 [AMD ROCm Installation Guide for Linux](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).
 
-Currently, the only combination tested is Ubuntu 18.04 with ROCm 4.1.0 using a Vega20 gfx906.
+Currently, the only combination tested is Ubuntu 18.04 with ROCm 4.2.0 using a Vega20 gfx906.
```

Comment:
It looks like https://github.com/intel/llvm/commit/bfbd5af782be92e1f741ab1e0ced594dc488322d invalidated this comment.
I suppose latest patches were tested on a different combination. 
@npmiller could you suggest the recommended combination of HW and SW to use for ROCM back-end?

Reply:
Sure, we have been testing on gfx906 and gfx908 with ROCm 4.2, so this change looks good to me.